docs/integrations/destinations/s3.md

```diff
@@ -128,13 +128,13 @@ For example:
 ```text
 testing_bucket/data_output_path/public/users/2021_01_01_1234567890_0.csv.gz
 ↑              ↑                ↑      ↑     ↑          ↑          ↑ ↑
-|              |                |      |     |          |          | format extension
-|              |                |      |     |          |          unique incremental part id
-|              |                |      |     |          milliseconds since epoch
-|              |                |      |     upload date in YYYY_MM_DD
-|              |                |      stream name
-|              |                source namespace (if it exists)
-|              bucket path
+|             |                                 |             |                           |                          |     | format extension |
+| undefined--------- | undefined----------------------------- | undefined--------- | undefined----------------------- | undefined---------------------- | u-d | ---------------- |
```

Comment:
This appears to be something the user should not be expecting, right? It seems like there's an autoformatter that believes these should be a table markdown but this is actually a rough around the edges path explanation diagram